packages/npm-dependencies-validation/test/utils/packageJsonUtil/packageJsonUtil.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import { VscodeFsUri, VscodeWsFolder } from "../../../src/types";
+import { isManagedByNpm6 } from "../../../src/utils/packageJsonUtil";
+
+describe("packageJsonUtil unit test", () => {
+  it("not manged by npm, has pnpm-workspace.yaml", async () => {
```

Comment:
add a test when it returns true

Reply:
for utilities it can be more thorough to do `.only` on current spec file and ensure 100% coverage for speciic tested file
via **only** it's matching ".spec" file.

But it is not mandatory